packages/ckeditor5-cloud-services/src/cloudservices.js

```diff
@@ -67,9 +76,61 @@ export default class CloudServices extends ContextPlugin {
 			return;
 		}
 
-		this.token = new CloudServices.Token( this.tokenUrl );
+		return this.registerTokenUrl( this.tokenUrl, true );
+	}
+
+	/**
+	 * Registers an additional authentication token URL for CKEditor Cloud Services or a callback to the token value promise. See the
+	 * {@link module:cloud-services/cloudservices~CloudServicesConfig#tokenUrl} for more details.
+	 *
+	 * @param {String|Function} tokenUrl The authentication token URL for CKEditor Cloud Services or a callback to the token value promise.
+	 * @param {Boolean} [isDefault=false] Whether this should be a default authentication token provider.
```

Comment:
It actually doesn't make sense to expose this flag as only the default value from the plugin initialization can use it.

So I'd try to remove the additional handling of the default token from this function.